Looms.PoS/Looms.PoS.Application/Models/Responses/Tax/TaxResponse.cs

```diff
@@ -0,0 +1,14 @@
+using Looms.PoS.Domain.Enums;
+
+namespace Looms.PoS.Application.Models.Responses.Tax;
+
+public class TaxResponse
```

Comment:
record

Reply:
no

Reply:
but like every other response is class?

Reply:
then every response should be changed to record :D

Reply:
changed it to record